src/pages/CreateGroup/CreateGroup.tsx

```diff
@@ -0,0 +1,153 @@
+import Row from "../../components/layout/Row";
+import { Link, useNavigate } from "react-router-dom";
+import LeftArrowIcon from "../../assets/arrow_left.svg";
+import RemoveIcon from "../../assets/close.svg";
+import React, { useState } from "react";
+import TextField from "../../components/TextField";
+import Column from "../../components/layout/Column";
+import Button from "../../components/Button";
+import { SimpleUser, User } from "../../data/models/User";
+import { useApi } from "../../contexts/ApiProvider";
+import ConfirmationModal from "../../components/modals/ConfirmationModal";
+import { Group } from "../../data/models/Group";
+import toast from "react-hot-toast";
+import { useUser } from "../../contexts/UserProvider";
+
+export default function CreateGroup() {
+  const { api } = useApi();
+  const { user } = useUser();
+  const navigate = useNavigate();
+
+  const [groupName, setGroupName] = useState("");
+  const [curUserAdd, setCurUserAdd] = useState("");
+  const [groupUsers, setGroupUsers] = useState<SimpleUser[]>([]);
+
+  const [nameError, setNameError] = useState("");
+  const [userAddError, setUserAddError] = useState("");
+
+  // Create group, invite users, navigate to group, and show toast
+  const createGroup = async () => {
+    const newGroup = await api.post<Group>("/group/", {
+      name: groupName,
+    });
+    await api.put<Group>("/group/" + newGroup.data.id, {
+      invite_users: groupUsers.map((u) => u.username),
+    });
+    navigate("/groups/" + newGroup.data.id);
+    toast.custom(
+      <div className="flex w-[400px] justify-center rounded-lg bg-light_yellow p-4">
+        <p className="p-0 font-semibold text-secondary">
+          Successfully Created Group
+        </p>
+      </div>,
```

Comment:
Could you make a toast message component for this? I'm assuming we will be using this type of toast throughout the app's lifespan. It could be very simple with just a label prop for the text.